apps/opik-backend/src/main/java/com/comet/opik/api/SpanBatch.java

```diff
@@ -0,0 +1,12 @@
+package com.comet.opik.api;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+import java.util.List;
+
+public record SpanBatch(@NotNull @Size(min = 1, max = 2000) @JsonView( {
```

Comment:
It makes sense; we can do it later.

However, I believe we should increase the query size and batch size since the queries are more straightforward now, and it will help with the ingestion.